packages/odie-client/src/components/message/messages-container.tsx

```diff
@@ -172,7 +172,7 @@ export const MessagesContainer = ( { currentUser }: ChatMessagesProps ) => {
 					aria-relevant="additions"
 				>
 					{ chat.messages.map( ( message ) => (
-						<div key={ message.created_at }>
+						<div key={ `${ message.created_at }-${ message.internal_message_id }` }>
```

Comment:
Wouldn't `internal_message_id` be enough to define a unique key?

```suggestion
						<div key={ message.internal_message_id }>
```